config.json

```diff
@@ -0,0 +1,232 @@
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": "DEBUG"
+    },
+    "Project": {
+      "description": "Project where the commands are going to be applied.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": ""
+    },
+    "ExtraArguments": {
+      "description": "Extra parameters will be passed to the selected command.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": ""
+    },
+    "BatchRestorePackages": {
+      "description": "MsBuild target that restores the packages.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanAllProjects": {
+      "description": "MsBuild target that deletes the binary output directory.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanPackages": {
+      "description": "MsBuild target that deletes the repo-local nuget package directory.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanPackagesCache": {
+      "description": "MsBuild target that deletes the user-local nuget package cache.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    }
+  },
+  "commands": {
+    "sync":{
+      "alias":{
+        "p":{
+          "description": "Restores all NuGet packages for repository.",
+          "settings":{
+            "RestoreDuringBuild":  true,
+            "BatchRestorePackages": "default"
+          }
+        },
+      },
+      "defaultValues":{
+        "toolName": "msbuild",
+        "settings": {
+          "MsBuildParameters":"default",
+          "MsBuildLogging":"default"
+        }
+      }
+    },
+    "build-managed":{
+      "alias":{
+        "s":{
+          "description": "Only builds binaries. It doesn't restore packages.",
+          "settings":{
+            "RestoreDuringBuild": false,
+            "BuildPackages": false,
+            "BuildTests": false
+          }
+        },
+        "packages":{
+          "description": "Builds the NuGet packages.",
+          "settings":{
+            "Project": "src\\packages.builds"
+          }
+        },
+        "debug":{
+          "description": "Sets ConfigurationGroup=Debug or the value passed by the user.",
+          "settings":{
+            "ConfigurationGroup": "Debug"
+          }
+        },
+        "release":{
+          "description": "Sets ConfigurationGroup=Release or the value passed by the user.",
+          "settings":{
+            "ConfigurationGroup": "Release"
+          }
+        },
+        "verbose":{
+          "description": "Passes /flp:v=normal to the msbuild command or the value passed by the user.",
+          "settings":{
+            "MsBuildLogging": "default"
+          }
+        },
+        "os":{
+          "description": "Sets OSGroup=AnyOS or the value passed by the user.",
+          "settings":{
+            "OSGroup": "default"
```

Comment:
Is default some keyword that indicates take the default value but also allows you to pass set it?